euphonic/qpoint_phonon_modes.py

```diff
@@ -716,3 +746,171 @@ def from_phonopy(cls: Type[T], path: str = '.',
             path=path, phonon_name=phonon_name, phonon_format=phonon_format,
             summary_name=summary_name)
         return cls.from_dict(data)
+
+    def write_phonon_website_json(
+        self,
+        output_file: str = "phonons.json",
+        name: str = "Euphonic export",
+        x_tick_labels: list[tuple[int, str]] | None = None,
+    ) -> None:
+
+        """Dump to .json for use with phonon website visualiser
+
+        Use with javascript application at
+        https://henriquemiranda.github.io/phononwebsite
+
+        Parameters
+        ----------
+        output_file
+            Path to output file
+        name
+            Set "name" metadata
+        x_tick_labels
+            index and label for high symmetry labels (if known)
+
+        """
+
+        with open(output_file, 'w') as fd:
+            json.dump(self._to_phonon_website_dict(name=name,
+                                                   x_tick_labels=x_tick_labels),
+                      fd)
+
+    @staticmethod
+    def _crystal_website_data(crystal: Crystal) -> dict[str, Any]:
+        elements = [
```

Comment:
No, Euphonic ships its own cross section data.

This package _is_ available on conda-forge and provides its own neutron cross-section data so it might be interesting to incorporate. I think it's overkill to add that dependency for this single lookup of element symbols, though.

Reply:
Yeah. Not for this PR for sure, but if you were using it, this would be a place to get around redefining things.